elasticdl_client/api.py

```diff
@@ -52,10 +52,9 @@ def init_zoo(args):
     tmpl_str = """\
 FROM {{ BASE_IMAGE }} as base
 
-RUN pip install elasticdl_preprocessing\
+RUN pip install elasticdl_preprocessing==0.2.0rc2 elasticdl==0.2.0rc2\
```

Comment:
I prefer not to specify the version number so that we can keep using the latest version.
Why do we install from mirror instead of pypi.org?

Reply:
We can use the fixed version in release branches. For develop branch, just keep the original implementation.

Reply:
> We can use the fixed version in release branches. For develop branch, just keep the original implementation.

I think it's a good idea, will close this PR.